### PR TITLE
Added RF State and SSA status bar to Cavity Display GUI

### PR DESCRIPTION
**Pull request by Derikka**

### CODE REVIEW

File: cavityDisplayGUI.py [hunk 1]
```diff
@@ -82,17 +81,29 @@ def __init__(self, parent=None, args=None):
...
-                rfStatusBarList = cmTemplateRepeater.findChildren(PyDMDrawingLine)
 
-                for cavityWidget, rfStatusBar in zip(cavityWidgetList, rfStatusBarList):
-                    cavityObject = cryomoduleObject.cavities[int(cavityWidget.cavityText)]
+                rfStatusBarList = []
+                ssaStatusBarList = []
+                statusBarList = cmTemplateRepeater.findChildren(PyDMDrawingLine)
+                for statusBar in statusBarList:
+                    if "RFSTATE" in statusBar.accessibleName():
+                        # print(statusBar.accessibleName())
```
Comment (lisazacarias) [inline cavityDisplayGUI.py]: Can we delete these comments?

File: utils.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 from csv import DictReader
 from typing import Dict, List
 
-DEBUG = False
-BACKEND_SLEEP_TIME = 8
+DEBUG = True
```
Comment (lisazacarias) [inline utils.py]: And can we flip this to False?

**Review by lisazacarias — CHANGES_REQUESTED** [review]

File: cavityDisplayGUI.py (hunk 1 above)
Comment (Derikka) [reply cavityDisplayGUI.py]: Done!

**Review by lisazacarias — APPROVED** [review]
Great!

**Pull request merged**